### PR TITLE
model output to log probability for the loss

### PR DESCRIPTION
**Pull request by cesc-park**
Model output should be changed to log probability

to calculate 

tf.nn.softmax_cross_entropy_with_logits(logpro, target)

### CODE REVIEW

**Comment by carpedm20** [comment]
@cesc-park I was wondering what is right and what is wrong. Did you closed this issue because the original code is right?

**Comment by cesc-park** [comment]
@carpedm20 Yep

softmax_cross_entropy_with_logits implicitly calculate log probability.

Tensorflow document
"tf.nn.softmax_cross_entropy_with_logits(logits, labels, name=None)
logits: Unscaled log probabilities."

Made me confusing.

Sorry!
